### ISSUE TITLE
Unable to use new script

### ISSUE THREAD

**Issue by apperrault** [aberg83/docker-server#5]
Hey
I think there is an issue with your script somewhere, I am not a Linux person so I can't be of much help.  Here are the symptoms that I am having

When prompted for the location on a new Ubuntu build i put
**Config Files**
/home/apperrault
**Media**
/home/apperrault/Videos
**Downloads**
/home/apperrault/Downloads

After running for a few minutes i get the following
![image](https://cloud.githubusercontent.com/assets/5666402/19695655/0705c594-9ab2-11e6-8714-9bc843eb6435.png)

After the pause I get this
![image](https://cloud.githubusercontent.com/assets/5666402/19695669/13652a8c-9ab2-11e6-9a95-af4cb2e1ea0d.png)

Any thoughts?  I really want to test this out on my VM to make sure I know how to set it up when I go for real

thanks
Aaron

**Comment by zjpleau** [comment]
I have the same issue. The path to the docker container config files is completely empty.

**Comment by aberg83** [comment]
are you guys copying and pasting the entire command?

**Comment by apperrault** [comment]
yeah.  I am copying and pasting the entire thing.  I am online for the next hour or so if there is anything you need me to try

app

**Comment by aberg83** [comment]
I'm trying an install on a fresh ubuntu 16.04. Will let you know what issues if any.

**Comment by apperrault** [comment]
OK.  I am a Linux Newb so maybe I am doing something wrong.  let me know if you want me to try something

app

**Comment by aberg83** [comment]
Ok, I fixed it. Try it now please.

**Comment by apperrault** [comment]
looks like it is getting farther.  Thanks for the help.

**Comment by apperrault** [comment]
OK, it completed!!!  How can I test the sites.  I currently have it running in a VM, but when I go to http://localhost/plex or even when i edit the /etc/hosts file to point my domain name to 127.0.0.1, I am getting page reset

**Comment by aberg83** [comment]
you need to forward ports 80 and 443 to your server IP on your router. and then restart the nginx container to generate the certificate.

before that, point your domain to your WAN IP.

then logout, log back in and run "docker logs -f nginx"

**Comment by aberg83** [comment]
Updated my post above with a better step by step approach to try.

**Comment by apperrault** [comment]
Thanks. I will try it in the morning. Go ahead and close the issue.

**Comment by zjpleau** [comment]
Alright I updated to the latest build. The script runs to completion but I still don't have anything when I browse. Confirmed my DNS is set up correctly (subdomain.domain.com/appname format). I am testing on a VPS directly on the internet so no port forwarding needed. Also confirmed no firewall running.

**Comment by aberg83** [comment]
What is the output of 'docker logs -f nginx'?

**Comment by zjpleau** [comment]
nginx: [emerg] BIO_new_file("/config/keys/letsencrypt/fullchain.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/config/keys/letsencrypt/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)

**Comment by aberg83** [comment]
The folder for your domain is missing. Did you enter a domain when asked? This is setup to work only with top level domains and www subdomains. For now at least.

**Comment by zjpleau** [comment]
Ahh ok, I think it's because I specified subdomain.domain.com instead of domain.com then.

**Comment by aberg83** [comment]
Let me know if it works for you once you use a TLD.

**Comment by apperrault** [comment]
So it installed and I can get to the Plex interface, but not any of the other sites. I rebooted the host and then restarted ngix but still I am getting a page reset when I try and navigate to the site.

**Comment by aberg83** [comment]
If you're comfortable creating a user account for me, I can poke around and see what's up.

**Comment by aberg83** [comment]
Also, paste output of 'docker logs -f nginx' and paste in your '$config/nginx/site-prefs/default' file. I'm sure analyzing those will give up the answer.

**Comment by zjpleau** [comment]
I'm still getting this error:
nginx: [emerg] BIO_new_file("/config/keys/letsencrypt/fullchain.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/config/keys/letsencrypt/fullchain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file)

Configured using TLD now.

**Comment by aberg83** [comment]
Sorry, it should be site-confs. What is your default file showing?

**Comment by zjpleau** [comment]
GNU nano 2.5.3                                                          File: docker/containers/nginx/nginx/site-confs/default                                                                                                                          

server {
        listen 80;

        listen 443 ssl default_server;

        root /config/www;
        index index.html index.htm index.php;

        server_name _;

        ssl_certificate /config/keys/letsencrypt/fullchain.pem;
        ssl_certificate_key /config/keys/letsencrypt/privkey.pem;
        ssl_dhparam /config/nginx/dhparams.pem;
        ssl_ciphers 'ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA25$
        ssl_prefer_server_ciphers on;

        client_max_body_size 0;

        location / {
                try_files $uri $uri/ /index.html /index.php?$args =404;
        }

        location ~ \.php$ {
                fastcgi_split_path_info ^(.+\.php)(/.+)$;
                # With php5-cgi alone:
                fastcgi_pass 127.0.0.1:9000;
                # With php5-fpm:
                #fastcgi_pass unix:/var/run/php5-fpm.sock;
                fastcgi_index index.php;
                include /etc/nginx/fastcgi_params;

**Comment by aberg83** [comment]
That looks like an untouched 'default' file. It should be removed by the script and then replaced with the generated file with all of the reverse proxy blocks.

I just updated the script and README and tested it on a fresh 16.04 install. Worked great with a top level domain name. No issues on my end. Please try again and post results/errors, etc.
